### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by RobHBOT**
Proposed addition of Pull Request Process

### CODE REVIEW

File: docs/governance/index.md [hunk 1]
```diff
@@ -36,3 +36,110 @@ All proposals can be found on the Hummingbot Foundation Snapshot: https://snapsh
 
 Voting is expected to commence in mid-to-late January 2022 with the Hummingbot 1.0 release. 
 
+## Pull Request Proposal (PRP)
```
Comment (phbrgnomo) [inline docs/governance/index.md:39]: Remove one # to differentiate the section

File: docs/governance/index.md [hunk 2]
```diff
@@ -36,3 +36,110 @@ All proposals can be found on the Hummingbot Foundation Snapshot: https://snapsh
...
+
+## What is a Pull Request Proposal (PRP)?
+HBOT token holders have the right to vote on which Pull Requests should be added to the main Hummingbot Client Release. The first step is the Pull Request Proposal. Anyone can create a Pull Request Proposal related to a Pull Request. There is no need to hold HBOT tokens to create a PR o PRP. The votation in snapshop will then determine what PRPs are approved by majority and subject to review by Foundation for determination of integrity, quality and security issues before merging to master branch.  
+All Pull Requests targeting the development branch will be merged into the main code-base only if there is an approved Pull Request Proposal on the Hummingbot Snapshot page related to it.
+
+## Hummingbot Development Cycle
+The Hummingbot Client Repository has three main branches related to the development cycle of each new version:
+
+## 1.	Development branch
```
Comment (phbrgnomo) [inline docs/governance/index.md:48]: Add one # to the numbered itens to differentiate the section

File: docs/governance/index.md [hunk 3]
```diff
@@ -36,3 +36,110 @@ All proposals can be found on the Hummingbot Foundation Snapshot: https://snapsh
...
+During the first month of the development cycle, Pull Request Proposals approved during this period will be scheduled to be added to the next month's release.
+Only Pull Request Proposals approved before the cut-off date will be added to the following release.
+
+Month 2
+During the second month of the development cycle, The Foundation team will merge all the remaining approved Pull Request Proposals, and move the development branch to the staging branch, and run all tests needed.
+A new release can happen on any day of the second month, as soon as all approved Pull Requests are merged, and the Foundation QA team signs off the release.
+
+## How to have your Pull Request added to the Hummingbot code base 
+
+## 1. Create the Pull Request
```
Comment (phbrgnomo) [inline docs/governance/index.md:81]: Add one # to the numbered itens to differentiate the section

File: docs/governance/index.md [hunk 4]
```diff
@@ -36,3 +36,110 @@ All proposals can be found on the Hummingbot Foundation Snapshot: https://snapsh
...
+For a Pull Request to be considered "open for voting", all the requirements below must be fulfilled:
+a.	Is targeted to the development branch;
+b.	Contains a clear description of what is the objective of the Pull Request;
+c.	Passes all the existing Unit Tests;
+d.	No merging conflicts exists;
+
+When all the requirements above are met, a Pull Request Proposal can be created, allowing the community to vote if the Pull Request will be merged into the development branch. If the Pull Request or the Proposal does not meet the requirements described above, it will be deleted by the Foundation Team
+
```
Comment (phbrgnomo) [inline docs/governance/index.md:89]: Add:
"NOTE: To test and improve the process of voting and merging the approved PRP, until January 30rd only Hummingbot Foundation members will be able to create new PRPs"

**Review by phbrgnomo — CHANGES_REQUESTED** [review]
Few edits needed.

File: docs/governance/index.md (hunk 1 above)
Comment (mifeng) [reply docs/governance/index.md:39]: # is the page-level heading. I suggest using ## for this one and ### for the sections under this.

**Review by phbrgnomo — APPROVED** [review]

**Review by phbrgnomo — APPROVED** [review]

**Review by mifeng — APPROVED** [review]

**Pull request merged**